src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -60,18 +59,19 @@
     (> new-value prev-value)))
 
 (defn- f-view-internal
-  ;; crypto-decimals and limit-crypto args are needed for component tests only
-  [{:keys [crypto-decimals limit-crypto]}]
+  [{:keys [crypto-decimals limit-crypto type]}]
   (let [bottom                    (safe-area/get-bottom)
+        navigate-back-route       (case type
+                                    :bridge :wallet-bridge-send
+                                    :wallet-send-input-amount)
         {:keys [currency]}        (rf/sub [:profile/profile])
         token                     (rf/sub [:wallet/wallet-send-token])
         loading-suggested-routes? (rf/sub [:wallet/wallet-send-loading-suggested-routes?])
         token-symbol              (:symbol token)
-        limit-crypto              (or limit-crypto
-                                      (utils/get-standard-crypto-format token (:total-balance token)))
+        limit-crypto              limit-crypto
         conversion-rate           (get-in token [:market-values-per-currency :usd :price])
         limit-fiat                (.toFixed (* (:total-balance token) conversion-rate) 2)
-        crypto-decimals           (or crypto-decimals (utils/get-crypto-decimals-count token))
+        crypto-decimals           crypto-decimals
```

Comment:
I think better keep crypto decimals calculation from `(utils/get-crypto-decimals-count token)` here because now it is being calculated in two places

Reply:
@Pau1fitz I still see it being called in two places, in `send_amount/view` and in `bridge_send/view`

> The reason for this was that the total-amount was not originally included
Which total-amount or total-value are you referring to?